seed/csharp-sdk/file-upload/src/SeedFileUpload/Core/QueryString.cs

```diff
@@ -0,0 +1,82 @@
+using System.Text.Json;
+using SeedFileUpload.Core;
+
+internal static class QueryString
```

Comment:
Discussed offline - couple options here.

1. Redefine these methods as extensions on the `Convert` class.
2. Update the name of this to be something like `QueryStringConverter`.